SPECS/kernel/config

```diff
@@ -857,38 +888,42 @@ CONFIG_BLK_DEV_INTEGRITY_T10=y
 CONFIG_BLK_DEV_ZONED=y
 CONFIG_BLK_DEV_THROTTLING=y
 # CONFIG_BLK_DEV_THROTTLING_LOW is not set
-# CONFIG_BLK_WBT is not set
+CONFIG_BLK_WBT=y
+CONFIG_BLK_WBT_MQ=y
```

Comment:
writeback throttling
https://www.spinics.net/lists/linux-btrfs/msg103581.html

Reply:
WBT aka 'writeback throttling' throttles background writes acording to
latency/throughput of the underlying block device in favor of readers.
It is meant to protect interactive/low-latency/desktop apps from heavy
bursts of background writeback activity. I tested early versions and
provided feedback to Jens Axboe; it really is helpful when it works,
but obviously cannot cater to every situation. There have been reports
that it is unhelpful for write-only/heavy workloads and may lead to
queueing pileup.

Reply:
looks useful for desktop stuff